models/text_recognition_crnn/README.md

```diff
@@ -38,10 +42,17 @@ python demo.py --input /path/to/image --model text_recognition_CRNN_CN_2021nov.o
 
 All files in this directory are licensed under [Apache 2.0 License](./LICENSE).
 
+## Examples
```

Comment:
The 'Examples' section should be put in `Demo` section under python commands and formatted as level-3 heading (`### Examples`).